src/Main.java

```diff
@@ -1,5 +1,55 @@
+import java.util.Arrays;
+
 public class Main {
-    public static void main(String[] args) {
-        System.out.println("Hello world!");
+    public static void main(String[] args)
+    {task1();
+
+    }
+    // Объявить метод «сгенерироватьМассив»
+    public static int[] generateRandomArray()
+    {
+        java.util.Random random = new java.util.Random();
+        int[] arr = new int[30];
+        for (int i = 0; i < arr.length; i++)
+        {
+            arr[i] = random.nextInt(100_000) + 100_000;
+        }
+        return arr;
+    }
+    public static void task1()
+    {
+        int [] arr=generateRandomArray();
+        //Задание 1
+        int summ=0;
+        for(int elmArr:arr){
+            summ+=elmArr;
+
+        }
+        System.out.println("Сумма трат за месяц составила "+summ+" рублей");
+        //Задание 2
+        int minZatr=arr[0];
+        int maxZatr=arr[0];
+        for(int elmArr:arr)
+        {
+            if(minZatr>=elmArr){
+                minZatr=elmArr;
+            }
+            if(maxZatr<=elmArr){
+                maxZatr=elmArr;
+            }
+
+
+        }
+        System.out.println("Минимальная сумма трат за день составила "+ minZatr+" рублей. " +
+                "Максимальная сумма трат за день составила "+maxZatr+" рублей");
+        //Ввиду того, что сумма элементов считается выше, то
+        //можно ее повторно не считать
+        float midZatr=summ/30;
```

Comment:
надо делить не на число, а на длину массива
```suggestion
        float midZatr=summ/arr.length;
```

Reply:
также деление целочисленное, т.е. средняя ЗП находится не совсем правильно, отбрасывается остаток